src/gui_server_plugin.cpp

```diff
@@ -95,23 +159,24 @@ void GUIServerPlugin::entityToMsg(const ed::EntityConstPtr& e, ed_gui_server::En
     }
 
     // HACK! Way of coding that this is a human
-    if (e->type() == "human" || e->hasFlag("possible_human"))
+    if (e->hasType("human") || e->hasType("person") || e->hasFlag("possible_human"))
```

Comment:
What is the difference between a `human` and a `person`. If there is already a `human` type, should we merge those 2 concepts?

Reply:
Might be a good thing. I can only find human in the perception plugins, which we don't use anymore.
`Possible_human` is used in laser_plugin. This feature might have become unnecessary.